commcare_connect/opportunity/tw_form_views.py

```diff
@@ -80,3 +94,166 @@ def verification_flags_config(request, org_slug=None, pk=None):
             form_json_formset=form_json_formset,
         ),
     )
+
+
+class OpportunityEdit(OrganizationUserMemberRoleMixin, UpdateView):
+    model = Opportunity
+    template_name = "tailwind/pages/opportunity_edit.html"
+    form_class = OpportunityChangeForm
```

Comment:
Since this is CBV, we can just subclass and override the `template_name` and `form_class` instead of full copy